templates/part.board.mainView.php

```diff
@@ -5,36 +5,39 @@
 		<p>{{ statusservice.text }}</p></div>
 </div>
 <div id="board-header">
-	<a href="#" title="<?php p($l->t('All Boards')); ?>">
-		<i class="icon icon-home"></i>
-	</a>
+	<div id="home" class="crumb svg last">
```

Comment:
hey @pixelipo If I use your example "home" icon is not visible (on NC12)
![screenshot_20170523_192633](https://cloud.githubusercontent.com/assets/16517095/26368255/6d3f3f36-3ff1-11e7-850f-702bed340e0b.png)